From ea7ffa46e37b8e0c0435974fc317016b209694f9 Mon Sep 17 00:00:00 2001
From: Ke Huang <keh@codeaurora.org>
Date: Thu, 11 Jan 2018 17:20:56 +0800
Subject: [PATCH] cnss: change pcie fw path for dual wifi support

change the pcie fw path on msm8996 vplatform for 11ac and pcie
dual wifi modules support.

Change-Id: I21882a439c2978813a631ac7ebc41c0ceba028d7
Signed-off-by: Ke Huang <keh@codeaurora.org>
---
 drivers/net/wireless/cnss/cnss_common.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/drivers/net/wireless/cnss/cnss_common.c b/drivers/net/wireless/cnss/cnss_common.c
index 0b73f37..3c68b58 100644
--- a/drivers/net/wireless/cnss/cnss_common.c
+++ b/drivers/net/wireless/cnss/cnss_common.c
@@ -42,9 +42,15 @@ static struct cnss_fw_files FW_FILES_QCA6174_FW_2_0 = {
 static struct cnss_fw_files FW_FILES_QCA6174_FW_1_3 = {
 	"qwlan13.bin", "bdwlan13.bin", "otp13.bin", "utf13.bin",
 	"utfbd13.bin", "epping13.bin", "evicted13.bin"};
+#ifdef CONFIG_MSM_GVM_QUIN
+static struct cnss_fw_files FW_FILES_QCA6174_FW_3_0 = {
+	"pcie/qwlan30.bin", "pcie/bdwlan30.bin", "pcie/otp30.bin", "pcie/utf30.bin",
+	"pcie/utfbd30.bin", "pcie/epping30.bin", "pcie/evicted30.bin"};
+#else
 static struct cnss_fw_files FW_FILES_QCA6174_FW_3_0 = {
 	"qwlan30.bin", "bdwlan30.bin", "otp30.bin", "utf30.bin",
 	"utfbd30.bin", "epping30.bin", "evicted30.bin"};
+#endif
 static struct cnss_fw_files FW_FILES_DEFAULT = {
 	"qwlan.bin", "bdwlan.bin", "otp.bin", "utf.bin",
 	"utfbd.bin", "epping.bin", "evicted.bin"};
-- 
1.9.1

